src/NServiceBus.Core.Tests/Timeout/InMemoryTimeoutPersisterThreadTests.cs

```diff
@@ -26,7 +26,7 @@ public void Run()
 
             for (var i = 0; i < 10; i++)
             {
-                var thread = new Thread(() => Runner(inMemoryTimeoutPersister, options));
+                var thread = new Thread(() => Runner(inMemoryTimeoutPersister, options).Wait());
```

Comment:
It is probably a none issue. But I don't like `Thread(() => Runner().Wait())`. We could just use `Task.Run`